visidata/loaders/_pandas.py

```diff
@@ -17,6 +17,57 @@ def open_dta(vd, p):
     if not getattr(VisiData, funcname, None):
         setattr(VisiData, funcname, lambda vd,p,ft=ft: PandasSheet(p.name, source=p, filetype=ft))
 
+@VisiData.api
+def save_dta(vd, p, *sheets):
```

Comment:
Save functions should be decorated with `@asyncthread`, as they could take an indefinite amount of time to save a large sheet.